### PR TITLE
Sample tabs

### PR DESCRIPTION
**Pull request by szok**
Connected to #1517

### CODE REVIEW

File: src/ralph/back_office/templates/backofficeasset/components.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <tbody>
+      {% for item in components %}
+        <tr>
+          <td>{{ item.label }}</td>
+          <td>{{ item.model }}</td>
+          <td>{{ item.serial_number }}</td>
+          <td>{{ item.speed }}</td>
+          <td>{{ item.size }}</td>
+          <td>{{ item.count }}</td>
+          <td>{{ item.action }}</td>
```
Comment (ghost) [inline src/ralph/back_office/templates/backofficeasset/components.html:26]: Could it be a a template tag which takes `cols` and `rows`?

Comment (szok) [inline src/ralph/back_office/templates/backofficeasset/components.html:26]: These are just samples, no logic, later it will look completely different.

**Comment by ghost** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

File: src/ralph/data_center/views.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        )
+        context['data_classification'] = [
+            {'label': 'Availability', 'value': 'A3'},
+            {'label': 'Confidentiality', 'value': 'C2'},
+            {'label': 'Integrity', 'value': 'I4'},
+        ]
+        context['logs'] = [
+            '/var/log',
+        ]
+        context['critical_pathes'] = [
```
Comment (vi4m) [inline src/ralph/data_center/views.py:97]: paths!

Comment (mkurek) [inline src/ralph/data_center/views.py:97]: should be patches here

**Pull request merged**